test/core/rules/test_cases/L015.yml

```diff
@@ -13,3 +13,13 @@ test_3:
 
 test_4:
   pass_str: SELECT DISTINCT (a + b) * c
+
+test_fix_1:
+  fail_str: |
+    select distinct(field_1)
+    from
+        my_table
+  fix_str: |
+    select distinct field_1
+    from
+        my_table
```

Comment:
Could you add tests to make sure https://docs.snowflake.com/en/sql-reference/functions/is-distinct-from.html isn't caught, and also https://www.postgresqltutorial.com/postgresql-select-distinct/?

Could you also add `fix_str`s to test_1 and test_2?

Reply:
Sure, I'll look into this.

Reply:
Great suggestion about adding `fix_str`! Test 2, which was passing before this PR, is creating an invalid "fix" that I had overlooked until just now.

Reply:
It appears that the Snowflake `IS DISTINCT FROM` syntax is not currently supported in SQLFluff, as this query fails to parse:
```
select c
    from t
    where c is distinct from c2
```

Reply:
Ok, I added a test case for the Postgres syntax.